src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/hmi/on_driver_distraction_notification.cc

```diff
@@ -126,6 +127,19 @@ void OnDriverDistractionNotification::Run() {
       static_cast<int32_t>(application_manager::MessageType::kNotification);
   (*on_driver_distraction)[strings::msg_params][mobile_notification::state] =
       state;
+  if ((*message_)[strings::msg_params].keyExists(
+          mobile_notification::lock_screen_dismissal_enabled)) {
+    const bool lock_screen_dismissal_enabled =
+        (*message_)[strings::msg_params]
+                   [mobile_notification::lock_screen_dismissal_enabled]
+                       .asBool();
+    application_manager_.set_lock_screen_dismissal_state(
```

Comment:
@IGapchuk one more cast bool to enum. Change it to value check and setup appropriate state.